tests/js/client/shell/shell-vst-dump.js

```diff
@@ -0,0 +1,67 @@
+/* jshint globalstrict:false, strict:false, maxlen: 200 */
+/* global arango, assertEqual */
+
+// //////////////////////////////////////////////////////////////////////////////
+// / @brief ArangoTransaction sTests
+// /
+// /
+// / DISCLAIMER
+// /
+// / Copyright 2018 ArangoDB GmbH, Cologne, Germany
+// /
+// / Licensed under the Apache License, Version 2.0 (the "License")
+// / you may not use this file except in compliance with the License.
+// / You may obtain a copy of the License at
+// /
+// /     http://www.apache.org/licenses/LICENSE-2.0
+// /
+// / Unless required by applicable law or agreed to in writing, software
+// / distributed under the License is distributed on an "AS IS" BASIS,
+// / WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// / See the License for the specific language governing permissions and
+// / limitations under the License.
+// /
+// / Copyright holder is triAGENS GmbH, Cologne, Germany
+// /
+// / @author Jan Steemann
+// //////////////////////////////////////////////////////////////////////////////
+
+const jsunity = require('jsunity');
+const arangodb = require('@arangodb');
+const db = arangodb.db;
+
+function vstDumpSuite () {
+  'use strict';
+
+  const cn = 'UnitTestsCollection';
```

Comment:
Since we involve getting and many complex external entities, the name "UnitTests" seems missleading.

Reply:
That is true, although I think a cleanup in this area should not be done in the context of this PR. Consider:
```
$ grep -r "UnitTests" tests/js/ | wc -l
4571
$ grep -r "UnitTestsCollection" tests/js/ | wc -l
635
```